res/controllers/midi-components-0.0.js

```diff
@@ -467,9 +467,19 @@
             // For the first messages, disregard the LSB in case
             // the first LSB is received after the first MSB.
             if (this.MSB === undefined) {
-                this.max = 127;
+                // a flaw in the Pot API does not mandate consumers
+                // to supply an accurate max value when using
+                // this with non-7-bit inputs. We cannot detect
+                // that in the constructor so set the max
+                // appropriately here
```

Comment:
Why can't that be detected in the constructor?

Reply:
Because we don't know how the component is being used until either the `input` handler (for regular 7-bit precision) or the `inputLSB/inputMSB` handlers (for 14-bit precision) have been called.

Reply:
Adding this requirement later is a breaking API change, so something we should reserve for the ES6 rework.